content/docs/2.10/operate/security.md

```diff
@@ -0,0 +1,15 @@
++++
+title = "Security"
+description = "Guidance to configure security options"
+weight = 100
++++
+
+## Use your own TLS Certificates
+
+KEDA uses self-signed certificates for different things. These certificates are generated and rotated by the operator, storing them in a Kubernetes secret (`kedaorg-certs`) that it's mounted in other KEDA components in the (default) path `/certs` using the names `tls.crt` and `tls.key` for TLS certificate and `ca.crt` and `ca.key` for CA certificate. KEDA also patches Kubernetes resources to include the `caBundle`, making Kubernetes to trust in the CA.
+
+While this is a good starting point, some end-users may want to use their own certificates which are generated from their own CA in order to improve security. This can be done disabling the certificate generation/rotation in the operator and updating default values in other components (if required). 
```

Comment:
```suggestion
While this is a good starting point, some end-users may want to use their own certificates which are generated from their own CA in order to improve security. This can be done by disabling the certificate generation/rotation in the operator and updating default values in other components (if required). 
```